packages/react-vapor/src/components/select/tests/MultiSelectWithPredicate.spec.tsx

```diff
@@ -150,5 +150,28 @@ describe('Select', () => {
                 expect(spy).toHaveBeenCalledWith(unselectListBoxOption(id, items[1].value));
             });
         });
+
+        describe('when predicates are processed on the server side', () => {

```

Comment:
Forgot to add a test for the onUpdate prop being called when `predicate` changes...